projects/1.16/assets/unnamed-animal-mod/unnamedanimalmod/lang/zh_cn.json

```
@@ -0,0 +1,151 @@
+  "entity.unnamedanimalmod.humphead_parrotfish": "隆头鹦嘴鱼",
+  "entity.unnamedanimalmod.musk_ox": "麝牛",
+  "entity.unnamedanimalmod.banana_slug": "香蕉蛞蝓",
+  "entity.unnamedanimalmod.marine_iguana": "海鬣蜥",
+  "entity.unnamedanimalmod.flashlight_fish": "灯眼鱼",
+  "entity.unnamedanimalmod.platypus": "鸭嘴兽",
+  "entity.unnamedanimalmod.platypus_egg": "鸭嘴兽蛋",
+  "entity.unnamedanimalmod.marine_iguana_egg": "海鬣蜥蛋",
+  "entity.unnamedanimalmod.elephantnose_fish": "象鼻鱼",
+  "entity.unnamedanimalmod.pacman_frog": "角蛙",
+  "entity.unnamedanimalmod.capybara": "水豚",
+  "entity.unnamedanimalmod.rocket_killifish": "火箭鱂鱼",
+  "entity.unnamedanimalmod.mangrove_snake": "黄环林蛇",
+  "entity.unnamedanimalmod.mangrove_boat": "红树船",
+  "entity.unnamedanimalmod.blubber_jelly": "端棍水母",
+  "entity.unnamedanimalmod.fiddler_crab": "招潮蟹",
+  "entity.unnamedanimalmod.mangrove_snake_egg": "黄环林蛇蛋",
+  "entity.unnamedanimalmod.leafy_seadragon": "叶海马",
+  "entity.unnamedanimalmod.spotted_garden_eel": "花园鳗",
+  "entity.unnamedanimalmod.mudskipper": "弹涂鱼",
+
+  "block.unnamedanimalmod.mangrove_door": "红树木门",
+  "block.unnamedanimalmod.mangrove_log": "红树原木",
+  "block.unnamedanimalmod.mangrove_planks": "红树木板",
+  "block.unnamedanimalmod.mangrove_button": "红树木按钮",
+  "block.unnamedanimalmod.mangrove_fence": "红树木栅栏",
+  "block.unnamedanimalmod.mangrove_fence_gate": "红树木栅栏门",
+  "block.unnamedanimalmod.mangrove_pressure_plate": "红树木压力板",
+  "block.unnamedanimalmod.mangrove_slab": "红树木台阶",
+  "block.unnamedanimalmod.mangrove_stairs": "红树木楼梯",
+  "block.unnamedanimalmod.mangrove_sapling": "红树树苗",
+  "block.unnamedanimalmod.mangrove_trapdoor": "红树木活板门",
+  "block.unnamedanimalmod.mangrove_wood": "红树木头",
+  "block.unnamedanimalmod.stripped_mangrove_log": "去皮红树原木",
+  "block.unnamedanimalmod.stripped_mangrove_wood": "去皮红树木头",
+  "block.unnamedanimalmod.salt_block": "盐块",
+  "block.unnamedanimalmod.mangrove_leaves": "红树树叶",
+  "block.unnamedanimalmod.flowering_mangrove_leaves": "开花红树树叶",
+  "block.unnamedanimalmod.mangrove_fruit": "红树果",
+
+  "block.unnamedanimalmod.rich_farmland": "肥沃农田",
+  "block.unnamedanimalmod.salt": "盐",
+
+  "block.unnamedanimalmod.mud_block": "泥块",
+  "block.unnamedanimalmod.mud_bricks": "泥砖",
+  "block.unnamedanimalmod.chiseled_mud_bricks": "錾制泥砖砖块",
+  "block.unnamedanimalmod.mud_brick_slab": "泥砖台阶",
+  "block.unnamedanimalmod.mud_brick_stairs": "泥砖楼梯",
+  "block.unnamedanimalmod.dried_mud_block": "干泥块",
+  "block.unnamedanimalmod.dried_mud_bricks": "干泥砖块",
+  "block.unnamedanimalmod.chiseled_dried_mud_bricks": "錾制干泥砖块",
+  "block.unnamedanimalmod.dried_mud_brick_slab": "干泥砖台阶",
+  "block.unnamedanimalmod.dried_mud_brick_stairs": "干泥砖楼梯",
+
+  "biome.unnamedanimalmod.mangrove_forest": "红树林",
+
+  "subtitle.unnamedanimalmod.frog.ambient": "青蛙呱呱叫",
+  "subtitle.unnamedanimalmod.frog.hurt": "青蛙受伤",
+  "subtitle.unnamedanimalmod.frog.death": "青蛙死亡",
+  "subtitle.unnamedanimalmod.southern_right_whale.ambient": "南露脊鲸鸣叫",
```

```suggestion
  "subtitle.unnamedanimalmod.southern_right_whale.ambient": "南露脊鲸：鸣叫",
```

依照原版 字幕格式为：
**来源：声音**

下同